docs/operation/upgrade.rst

```diff
@@ -24,14 +24,40 @@ Upgrade Pre-requisites
 Before proceeding with the upgrade procedure, make sure to complete the
 pre-requisites listed in :doc:`/operation/preparation`.
 
+Run pre-check
+-------------
 You can test if your environment will successfully upgrade with the following
 command.
-This will simulate the upgrade procedure and provide an overview of the
+This will simulate the upgrade prechecks and provide an overview of the
 changes to be carried out in your MetalK8s cluster.
 
+.. important::
+
+    The version prefix metalk8s-**X.X.X** as used below during a MetalK8s
+    upgrade must be the new MetalK8s version you would like to upgrade
+    to.
+
    .. code::
 
-     /srv/scality/metalk8s-X.X.X/upgrade.sh --destination-version <destination_version> --dry-run --verbose
+     /srv/scality/metalk8s-X.X.X/upgrade.sh --destination-version \
+       <destination_version> --dry-run --verbose
+
+Backup old credentials
+----------------------
+Starting 2.5.0, MetalK8s will henceforth implement OpenID Connect (OIDC) based
+authentication for both the MetalK8s UI and Grafana UI. Both UIs now make use
+of a common user store and ofcourse a common access credential.
+
+.. warning::
+
+    To run an upgrade to version 2.5.0, MetalK8s administrators **must**
+    perform a manual backup/save the passwords set for MetalK8s UI and Grafana
```

Comment:
We can't really "backup" anything, they are passwords. Once set, they are encrypted, so no way to put those back into a new system. You need to "remember" them :innocent: 

Maybe rephrase to:
```suggestion
    ensure all static users defined in `/etc/htpasswd` can be recreated,
    and if any, all users that were defined in Grafana.
```

Reply:
Done